engines/uqm-megamod/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# CLONE PHASE
+git clone https://github.com/JHGuitarFreak/UQM-MegaMod.git source
```

Comment:
Are you also needing to clone and use https://github.com/JHGuitarFreak/UQM-MegaMod-Content/?

Reply:
Ah ok, does the repo contain everything for all of those mod files or just parts of them?